neuromation/cli/utils.py

```diff
@@ -612,7 +615,13 @@ def pager_maybe(
 
 
 def steal_config_maybe(dst_path: pathlib.Path) -> None:
```

Comment:
Drop this function to fail fast if an outdated dependency tries to import the function.

Hmm, looks like `Factory.get()` should steal config by default to make the functionality usable.
Otherwise, all `api.get()` callers should care about handling `pass_config` and `login_with_passed_config()` explicitly.

Reply:
OK, got it.
Then, do we need `login_with_passed_config` at all? If `Factory.get()` steels config by default, is see no reason for someone to call this function.
Also by implementing it this way (only inside `.get`), we can hide from user that we use ENV variable -- he/she still will be aware of it because of validation, but it will be impossible to write code that depends on this fact.